Dockerfile

```diff
@@ -307,28 +292,6 @@ RUN set -ex; \
         done; \
     fi
 
-# pgvector is an open-source vector similarity search for Postgres
-# https://github.com/pgvector/pgvector
-# As pgvector assumes running on the same system it was build at,
-# and this is not always our case (since AWS has instances with
-# different types of CPU, disable CPU-specific optimizations by
-# supplying OPTFLAGS="".
-# See https://github.com/pgvector/pgvector/issues/143
-# TODO: switch to debian packages once v0.50.0 is available there.
```

Comment:
💯 , thanks for getting back to this TODO! As of v0.5.1 the OPTFLAGS=native is commented:
```
/pgvector-0.5.1/debian/patches# cat no-native
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@
 REGRESS = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = --inputdir=test --load-extension=$(EXTENSION)

-OPTFLAGS = -march=native
+#OPTFLAGS = -march=native
```

Reply:
pgdg is our friend :D